Gopkg.toml

```diff
@@ -20,34 +20,34 @@
 #  name = "github.com/x/y"
 #  version = "2.4.0"
 
+[prune]
+  unused-packages = true
+  go-tests = true
+
 [[constraint]]
-  name = "github.com/Sirupsen/logrus"
-  revision = "8fcafe25b7418beae1d0e6875bde08c6411e8960"
+  name = "github.com/sirupsen/logrus"
+  revision = "8ab1e1b91d5f1a6124287906f8b0402844d3a2b3"
   source = "github.com/gravitational/logrus"
 
-[[constraint]]
-  name = "github.com/ghodss/yaml"
 
 [[constraint]]
   name = "github.com/gravitational/trace"
-  revision = "a81b26cd14cab7e5045b9391fe77662c00a7e438"
+  branch = "master"
 
 [[constraint]]
-  name = "gopkg.in/alecthomas/kingpin.v2"
+  branch = "release-1.9"
+  name = "k8s.io/apiextensions-apiserver"
 
-[[constraint]]
-  name = "gopkg.in/check.v1"
+[[override]]
+  name = "k8s.io/api"
+  branch = "release-1.9"
 
-[[constraint]]
-  branch = "release-1.7"
+[[override]]
   name = "k8s.io/apimachinery"
+  revision = "5134afd2c0c91158afac0d8a28bd2177185a3bcc"
```

Comment:
Do we need a particular revision here, or can use `release-1.9` branch?

Reply:
The release-1.9 branch wasn't compatible when I tried it yesterday, and iirc this is what's currently in the glide.lock file.